fiftyone/server/routes/embeddings.py

```
@@ -39,7 +41,16 @@ class OnPlotLoad(HTTPEndpoint):
     @route
     async def post(self, request: Request, data: dict) -> dict:
         """Loads an embeddings plot based on the current view."""
-        return await run_sync_task(self._post_sync, data)
+        try :
+            return await run_sync_task(self._post_sync, data)
+        except Exception as e:
+            msg = f"Unknown error occurred."
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)